workflow/doc.go

```diff
@@ -266,6 +266,27 @@ The workflow.ExecuteChildWorkflow() function is very similar to the workflow.Exe
 patterns described for using the workflow.ExecuteActivity() apply to the workflow.ExecuteChildWorkflow() function as
 well.
 
+Child workflows can also be configured to continue to exist once their parent workflow is closed. When using this
+pattern, extra care needs to be taken to ensure the child workflow is started before the parent workflow finishes.
+
+	cwo := workflow.ChildWorkflowOptions{
+		// Do not specify WorkflowID if you want cadence to generate a unique ID for child execution
+		WorkflowID:                   "BID-SIMPLE-CHILD-WORKFLOW",
+		ExecutionStartToCloseTimeout: time.Minute * 30,
```

Comment:
Remove WorkflowID and ExecutionStartToCloseTimeout fields as they are not relevant to the example.